deploy/operator.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: mattermost-operator
       containers:
         - name: mattermost-operator
-          image: mattermost/mattermost-operator:latest
+          image: mattermost/mattermost-operator:test
```

Comment:
Can we leave this on latest?

Reply:
no because this is being used for the e2e tests. can we change that to use latest instead of test. or sed the file to replace `latest` to `test` during the e2e @jwilander

Reply:
Yeah, this should be `:test`. Some more info about it here:

https://github.com/mattermost/mattermost-operator/pull/32#issuecomment-500937414

Reasoning for it here:

https://github.com/operator-framework/operator-sdk/blob/master/doc/test-framework/writing-e2e-tests.md#running-the-tests

Reply:
Ok cool, I'll just add a sed to the makefile command